.storybook/pages/StudentRefinement/StudentRefinement.stories.ts

```diff
@@ -0,0 +1,12 @@
+import type { StoryObj, Meta } from '@storybook/react';
+
+import { StudentRefinement } from './StudentRefinement';
+
+export default {
+  title: 'Pages/SDT/StudentRefinement',
+  component: StudentRefinement,
+} as Meta<Args>;
```

Comment:
Can we get a mobile view of this page?

Reply:
https://user-images.githubusercontent.com/86632227/199840873-5b7c87f8-2978-4be4-a1f4-d1921a658154.mov

Reply:
Sorry, I meant can we get a chromatic test covering the mobile view of this page.

Reply:
Oh ya, definitely. Added some chromatic breakpoints